src/blas/backends/portblas/portblas_level1.cxx

```diff
@@ -370,10 +370,9 @@ sycl::event rotmg(sycl::queue &queue, real_t *d1, real_t *d2, real_t *x1, real_t
                   const std::vector<sycl::event> &dependencies) {
     auto y_d =
         (real_t *)sycl::malloc_device(sizeof(real_t), queue.get_device(), queue.get_context());
-    auto copy_in_event = queue.memcpy(y_d, &y1, sizeof(real_t), dependencies);
+    queue.memcpy(y_d, &y1, sizeof(real_t), dependencies).wait();
```

Comment:
Can you add a comment here to explain this `memcpy` is a work around and link to the intel/llvm issue?

Reply:
Thank you for the review @Rbiessy.
I added a comment in 7f2c805